test/unit/master/test_build.py

```diff
@@ -429,6 +430,15 @@ def test_get_failed_atoms_returns_failed_atoms_only(self):
             build._all_subjobs_by_id[3]._atoms[3],
         ])
 
+    def test_delete_temporary_build_artifact_files_skips_results_tarball(self):
+        build = self._create_test_build(BuildStatus.BUILDING)
```

Comment:
I mean like 
`build = self._create_test_build(BuildStatus.FINISHED, ...`
instead of 
`build = self._create_test_build(BuildStatus.BUILDING, ...`